go.mod

```diff
@@ -46,3 +46,5 @@ require (
 	google.golang.org/genproto/googleapis/api v0.0.0-20240513163218-0867130af1f8 // indirect
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20240513163218-0867130af1f8 // indirect
 )
+
+retract v0.4.7 // Wait with locked mutex leads to the deadlock
```

Comment:
what does this do?

Reply:
I don't think I can explain better that [Go](https://go.dev/ref/mod#go-mod-file-retract) devs [can](https://play-with-go.dev/retract-module-versions_go119_en/). But basic idea here is to say that this version should't be used and explain the reason. The tooling knows about this directive and will skip this version.